---
marp: true
paginate: true
transition: glow pivot rotate swipe
style: |
  section::after {
    content: attr(data-marpit-pagination) '/' attr(data-marpit-pagination-total);
  }

---

# <!-- fit --> Trial 2 - [US patent 7279327B2: _Methods for producing recombinant coronavirus_](https://patents.google.com/patent/US7279327B2/en)

Welcome to July 2021, and meet today's protagonist: _dad_ üë®üèΩüëçüèΩ, also known as **SARS-CoV-1**.

With **SARS-CoV-2**, they make up **the same** [SARSr-CoV](https://en.wikipedia.org/wiki/SARS-related_coronavirus) **species**, and are about 79% genetically identical (_make the same proteins_). Putting the number in perspective:

:one: Two random humans are 99.8-99.9% identical üë®üèΩüë®üèø
:two: A human and a chimp: 98% üë®üèΩüêµ
:three: A human and a mouse: 85% üë®üèΩ:mouse:

...and then you have our two coronaviruses at 79% üëæ:alien:

---

## <!-- fit -->Heard you like reading, so we put some reading in your reading

:us: _Citizen's Journal_ opens its [_Patents Prove SARS-CoV-2 Is A Manufactured Virus_](https://www.citizensjournal.us/patents-prove-sars-cov-2-is-a-manufactured-virus/) with [a list of every SARS-CoV-1-related patent](https://www.m-cam.com/wp-content/uploads/2020/04/20200403_SARS_CoV_Patent_Corpus_Lit_Review.pdf) known to man :books::closed_book::books::skull:

![Long patent list is long](https://user-images.githubusercontent.com/13955209/180362167-acb11ccb-84f2-41c2-b8aa-057299192b51.gif) ![](https://user-images.githubusercontent.com/13955209/180490661-0a8d2e02-5150-4aeb-9888-b3cb037a1185.png)

---

## Whittling it down :knife::seedling::deciduous_tree::deciduous_tree:

Luckily, the article does go on to narrow the list down a bit. If it's your first run-in with the biotechnology patent world, you're probably in for some :zap: shock:

- [Canine coronavirus S[pike] gene and uses therefor](https://patents.google.com/patent/US6057436A/en)
    - Filed for by Pfizer in 1993 :arrow_right: granted 2000
    - the first contact with a spiked coronavirus had taken place in the 1970s, when military sentry :dog::dog: (:de: shepherds) suffered a violent diarrhea epidemic
    - suggests spike injection into other viruses, for targetting other species :grin::+1::+1:
- [Coronavirus isolated from humans](https://patents.google.com/patent/US7220852B1/en)
    - Filed by Centers of Disease Control and Prevention in 2004 :arrow_right: granted 2007
    - patents a naturally occurring, living thing (took a few tries) :woman_shrugging:

---

## Settling on a hypothesis :ballot_box_with_check:

The [Methods for producing recombinant coronavirus (US7279327B2)](https://patents.google.com/patent/US7279327B2/en) patent looks like something we **could verify**, using a modern SARS-CoV-2 genome. How come?

	Spike	
	 3	
	Envelope	
	Membrane	
	 6	
	 7	
	 8	
	Nucleocapsid	
	3' UTR	

---

FIG. 1D depicts the strategy for assembling recombinant TGEV and replicon cDNAs. The six cDNA subclones (TGEV A, B1, B2, C, DE1, and F deletion fragments) spanning the genome are flanked by unique interconnecting BglI and BstXI sites, allowing for directional assembly into a full-length replicon cDNA by in vitro ligation. TGEV A contained a unique T7 start site at the 5‚Ä≤ end and the deletion fragments (FiGFP2-AvrII and FiGFP2-EcoNI) contain GFP and a 25-nt T tail, allowing for the synthesis of capped T7, polyadenylated transcripts in vitro.

Clones were identified by DNA sequencing using an ABI model 377 automated sequencer and constructs TGEV pFiGFP2(PflMI) and TGEV iGFP2(ScaI) were subsequently used in the assembly of recombinant TGEV viral cDNA and as the backbone for the construction of structural gene deletions (FIG. 1D).

The six cDNA subclones spanning the entire TGEV genome, including the FiGFP2(PflMI) and FiGFP2(PflMI) deletion subclones, were used to assemble TGEV viral and replicon constructs, respectively, as previously described (FIG. 1D) (Yount, et al. (2000) J. Virol. 74:10600-10611; Curtis, et al. (2002) J. Virol. 76:1422-1434).

---

FIG. 4A depicts the structure of constructs expressing recombinant PRRS GP5.

Previous studies have demonstrated that the PRRSV M protein accumulates in the ER of infected cells where it forms disulfide-linked heterodimers with GP5. Heterodimer formation may be critical in eliciting neutralizing antibody against conformational epitopes (Balasuriya, et al. (2000) J. Virol. 74:10623-30). Using the TGEV 3F subclone (FIG. 4A), GFP was removed by ClaI/PflMI digestion and replaced with GP5 of PRRSV to create icTGEV PRSS GP5 recombinant viruses (FIG. 4A).

GFP was removed by ClaI/PflMI digestion and replaced with GP5 of PRRSV to create icTGEV PRSS GP5 recombinant viruses (FIG. 4A).

---

FIG. 5 depicts the structure of the TGEV N gene rearranged viruses. The TGEV N gene and TSE were inserted downstream of the 3A TSE site by removing the ORF 3A (TGEV 2N(PflMI)) or ORF 3A and a part of ORF 3B (TGEV 2N(ScaI)). The downstream N gene was flanked by restriction sites and removed to produce the SNEM rearranged viruses.

The results provided above suggest that flanking sequences enhance transcription from the N TSE element. To test this hypothesis, GFP was replaced with the TGEV N gene and N TSE site as shown in FIG. 5. TGEV recombinant viruses were isolated that contained two copies of the N gene (TGEV 2N) as well as TGEV SNEM rearranged viruses that lack the ‚Äúnatural N orientation, and express N from the ORF3 position (FIG. 5).

2N viruses

---

_Introduce Occam's razor_

